WORKSPACE

```diff
@@ -403,9 +403,9 @@ gapic_generator_csharp_repositories()
 ##############################################################################
 # Ruby
 ##############################################################################
-_gapic_generator_ruby_version = "e10d40afa96a28036da03bb9b0af17d702715886"
+_gapic_generator_ruby_version = "2b66e7aca8d5d7d4cb7bf436776d7713d264cab8"
```

Comment:
just in case: this is the newer version, rigth?

Reply:
Yes - this [matches](https://github.com/googleapis/googleapis/blob/master/WORKSPACE#L361) the latest hash that is in googleapis.